third_party/terraform/website/docs/r/storage_bucket.html.markdown

```diff
@@ -39,6 +39,43 @@ resource "google_storage_bucket" "image-store" {
 }
 ```
 
+```hcl
+resource "google_storage_bucket" "image-store" {
+  name     = "image-store.com"
+  location = "US"
+  force_destroy = true
+
+  bucket_policy_only = true
+
+  website {
+    main_page_suffix = "index.html"
+    not_found_page   = "404.html"
+  }
+  cors {
+    origin = ["http://image-store.com"]
+    method = ["GET", "HEAD", "PUT", "POST", "DELETE"]
+    response_header = ["*"]
+    max_age_seconds = 3600
+  }
+ }
+```
+
+```hcl
+resource "google_storage_bucket" "image-store" {
+  name     = "image-store-bucket"
+  location = "US"
+  force_destroy = true
+
+  lifecycle_rule {
+    condition {
+     age = "3"
```

Comment:
User facing terraform configs should be correctly formatted. You can run `terraform fmt` on these examples to fix the spacing to be consistent.

Reply:
@chrisst `terraform fmt` not working for html.markdown files. I used html preview to verify the format, however it would be great if i can use the command to fix the format. I need some help to get the format command working

Reply:
Format fixed. Thank you for the tip @chrisst